common/plugins/ipam.go

```diff
@@ -39,7 +39,7 @@ type Route struct {
 	Nexthop *net.IP
 }
 
-// Sort an array of routes by mask, narrow first
+// ByMask sort an array of routes by mask, narrow first.
```

Comment:
Sorry, I meant `// ByMask is used to sort an array of routes by mask, narrow first.`